test/integration/utils.go

```diff
@@ -0,0 +1,350 @@
+package k8s
+
+import (
+	"context"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/Azure/azure-container-networking/test/internal/k8sutils"
```

Comment:
it seems silly to have a `utils.go` in package `k8s`, and a package `k8sutils` (and that name is just an antipattern). maybe it would be more appropriate to move this functionality in to that package (potentially rename to `kubernetes`) and consolidate all our test infra k8s machinery in one place?

Reply:
@rbtr  What's better, naming it something besides `kubernetes`, or importing it under a differnet name like `acnk8s` ?
![image](https://github.com/Azure/azure-container-networking/assets/35265851/92f4694b-e5b8-4a19-8638-6bb2d4d0e482)

Reply:
import it under a different name

Reply:
It's actually just this one file `datapath_win.go` so I'll just name it `acnk8s` in this import only